tests/Dockerfile

```diff
@@ -0,0 +1,3 @@
+FROM golang:1.12.5
+
+RUN apt-get update && apt-get install -y libtool libtool-bin
```

Comment:
IIUC this is used to build [`kubevirt/golang-libtool`](https://hub.docker.com/r/kubevirt/golang-libtool) , which in turn is used by [the KubeVirt CI](https://github.com/kubevirt/project-infra/pull/153/commits/84c67580c1971582f6096a982abc9ddb6d0511b1#diff-afcd5e274df06f6e9df04adb41efbbb3R8).

This makes sense, but I don't understand how the Dockerfile is turned into a container image. Do you have some tooling picking it up and submitting the build on Docker Hub? Is it built and uploaded manually?

Another partially related question: the other CI jobs are using the vanilla `golang:1.12.5` image instead of the custom one... Wouldn't it make sense to run all Virt Blocks jobs in the same container image for consistency?

Reply:
> This makes sense, but I don't understand how the Dockerfile is turned into a container image. Do you have some tooling picking it up and submitting the build on Docker Hub? Is it built and uploaded manually?

It is right now just manually. A solution is to have a postsubmit job which pushes it, in case it got changed.

> Another partially related question: the other CI jobs are using the vanilla `golang:1.12.5` image instead of the custom one... Wouldn't it make sense to run all Virt Blocks jobs in the same container image for consistency?

Yes makes probably sense.

Reply:
> > This makes sense, but I don't understand how the Dockerfile is turned into a container image. Do you have some tooling picking it up and submitting the build on Docker Hub? Is it built and uploaded manually?
> 
> It is right now just manually. A solution is to have a postsubmit job which pushes it, in case it got changed.

Alright, we can look into that down the line - I just wanted to make sure I understood the entire pipeline.

> > Another partially related question: the other CI jobs are using the vanilla `golang:1.12.5` image instead of the custom one... Wouldn't it make sense to run all Virt Blocks jobs in the same container image for consistency?
> 
> Yes makes probably sense.

Do you want to take care of that? Otherwise I can do it myself - this kind of change should be within my abilities O:-)

Reply:
> Do you want to take care of that? Otherwise I can do it myself - this kind of change should be within my abilities O:-)

Please go ahead! :)

Reply:
Done: https://github.com/kubevirt/project-infra/pull/154